File: Typescript/calculator.ts
import { createInterface } from "readline/promises";
function doTheMath(f: string, o: string, s: string) {
  let output: number | string = "";
  switch (o) {
    case "*":
      output = parseFloat(f) * parseFloat(s);
      break;

    case "/":
      output = parseFloat(f) / parseFloat(s);
      break;
    case "+":
      output = parseFloat(f) + parseFloat(s);
      break;
    case "-":
      output = parseFloat(f) - parseFloat(s);
      break;
    default:
      output = "Invalid operation";
  }
  return output;
}
(async () => {
  const readline = createInterface({
    input: process.stdin,
    output: process.stdout,
  });
  const first_number = await readline.question("Enter first number:");
  const operation = await readline.question("Enter operation:(+, -, *, /):");
  const second_number = await readline.question("Enter second number:");
  const output = doTheMath(first_number, operation, second_number);
  console.log("Result:", output);
  readline.close();
})();

File: Python/calculator.py
def do_the_math(f,o,s):
    match (o):
        case "*":
            output = float(f) * float(s)
        case "/":
            output = float(f) / float(s)
        case "+":
            output = float(f) + float(s)
        case "-":
            output = float(f) - float(s)
    return output

if __name__ == "__main__":
    first_number  =input('Enter first number:')
    operation  =input('Enter operation:(+, -, *, /):')
    second_number  =input('Enter second number:')
    output = do_the_math(first_number, operation, second_number)
    print("Result:", output)